sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -512,6 +552,9 @@ public EncryptedBlobClientBuilder serviceVersion(BlobServiceVersion version) {
      * <p>If {@code pipeline} is set, all other settings are ignored, aside from {@link #endpoint(String) endpoint} and
      * {@link #serviceVersion(BlobServiceVersion) serviceVersion}.</p>
      *
+     * <p>Note that this method does not copy over the {@link CustomerProvidedKey} and encryption scope properties
```

Comment:
Can you add "for security reasons" to hopefully preempt some customers who may ask for this?

Reply:
done